packages/replayio/src/utils/spawnProcess.ts

```diff
@@ -54,6 +54,11 @@ export function spawnProcess(
 
     spawned.on("exit", (code, signal) => {
       if (code || signal) {
+        // Don't fail on manual closing
+        if (signal === "SIGTERM") {
```

Comment:
I don't get `SIGTERM` re-raised by our Chromium here. When I close the browser by pressing any key in the terminal I get `code === 0 && signal === null` here. This should only depend on the Chromium's behavior... so I wonder why it's inconsistent 🤔

Reply:
That's interesting. FWIW it makes sense to me that it throws a SIGTERM given that the `kill` command reads as:

> The signal to send, either as a string or number. Default: 'SIGTERM'.

https://nodejs.org/api/process.html#processkillpid-signal

TBH I have only tested this on Windows, not macOS

Reply:
> I don't get `SIGTERM` re-raised by our Chromium here. When I close the browser by pressing any key in the terminal I get `code === 0 && signal === null` here. This should only depend on the Chromium's behavior... so I wonder why it's inconsistent 🤔

Same, FWIW.